### PR TITLE
chore: optimize image size

### PR DESCRIPTION
**Pull request by yqrashawn**
compress image with https://github.com/kornelski/pngquant

size change

|                                                   |                          |
|:--------------------------------------------------|:-------------------------|
| public/imgs/faucet/arrow.png                      | Bin 461 -> 328 bytes     |
| public/imgs/faucet/error.png                      | Bin 3245 -> 1734 bytes   |
| public/imgs/faucet/info.png                       | Bin 1299 -> 656 bytes    |
| public/imgs/faucet/left-arrow.png                 | Bin 316 -> 286 bytes     |
| public/imgs/faucet/right-arrow.png                | Bin 314 -> 242 bytes     |
| public/imgs/home/discord.png                      | Bin 17769 -> 11165 bytes |
| public/imgs/home/github.png                       | Bin 14902 -> 9750 bytes  |
| public/imgs/home/logo.png                         | Bin 16380 -> 4099 bytes  |
| public/imgs/home/more.png                         | Bin 531 -> 377 bytes     |
| public/imgs/home/note.png                         | Bin 15088 -> 8196 bytes  |
| public/imgs/homepage/blog/kzg/zkrollup.png        | Bin 78179 -> 30151 bytes |
| public/imgs/login/discord.png                     | Bin 17769 -> 11165 bytes |
| public/imgs/login/github.png                      | Bin 14902 -> 9750 bytes  |
| public/imgs/login/twitter.png                     | Bin 13368 -> 7204 bytes  |
| public/imgs/media/discord.png                     | Bin 17769 -> 11165 bytes |
| public/imgs/media/github.png                      | Bin 14902 -> 9750 bytes  |
| public/imgs/media/twitter.png                     | Bin 13368 -> 7204 bytes  |
| public/imgs/token/gho.png                         | Bin 5008 -> 809 bytes    |
| public/imgs/token/weth.png                        | Bin 1496 -> 756 bytes    |
| public/logo_walletconnect.png                     | Bin 22732 -> 9619 bytes  |
| public/og_scroll.png                              | Bin 9892 -> 3845 bytes   |
| src/assets/images/homepage/home/more.png          | Bin 531 -> 377 bytes     |
| src/assets/images/homepage/home/start_bridge.png  | Bin 1447 -> 772 bytes    |
| src/assets/images/homepage/home/start_link.png    | Bin 1126 -> 666 bytes    |
| src/assets/images/homepage/home/start_setting.png | Bin 2291 -> 1119 bytes   |

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205536869882120

### CODE REVIEW

**Comment by Holybasil** [comment]
according to #786 , some images have been removed. we can merge #786 first, what do you think? @zzq0826 
and for image, can we prioritize using svg as much as possible?

**Comment by zzq0826** [comment]
> according to #786 , some images have been removed. we can merge #786 first, what do you think? @zzq0826 and for image, can we prioritize using svg as much as possible?

Let's merge this PR after merging #786.
yeah, agreed. For images of the icon/logo, let's try to use SVG as much as possible.

**Review by Holybasil — APPROVED** [review]

**Pull request merged**
